services/converters.go

```diff
@@ -150,7 +161,21 @@ func ToAPIAgent(q *reform.Querier, agent *models.Agent) (inventorypb.Agent, erro
 		return nil, err
 	}
 
-	// agents without services
+	var nodeID, serviceID string
```

Comment:
This is an exported function so it is possible to call it with agent=nil.
Probably we should check that case

Reply:
The idea is that it should not be called with nil – and it will panic if nil is passed.